frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -36,6 +38,43 @@ export function DashboardInfoSidebar({
   dashboard,
   setDashboardAttribute,
 }: DashboardInfoSidebarProps) {
+  const [page, setPage] = useState<"default" | "caching">("default");
+
+  return (
+    <DashboardInfoSidebarRoot
+      style={{ padding: page === "default" ? "0 2rem 0.5rem" : "1rem 0 0 0" }}
+      data-testid="sidebar-right"
+    >
+      <ErrorBoundary>
+        {page === "default" && (
+          <DashboardInfoSidebarBody
+            dashboard={dashboard}
+            setDashboardAttribute={setDashboardAttribute}
+            setPage={setPage}
+          />
+        )}
+        {page === "caching" && (
+          <PLUGIN_CACHING.DashboardStrategySidebar
+            dashboard={dashboard}
+            setPage={setPage}
+          />
+        )}
```

Comment:
❤️  i like the idea of adding a sub-page to this sidebar rather than wading into the mess of sidebar swapping